mcs/class/corlib/System.Reflection/Module.cs

```diff
@@ -446,5 +436,7 @@ public virtual Type[] GetTypes()
 		public virtual IEnumerable<CustomAttributeData> CustomAttributes {
 			get { return GetCustomAttributesData (); }
 		}
+
+		internal abstract IntPtr GetImpl ();
```

Comment:
Could you please explain why?

Reply:
internal abstract is not part of the contract but still required by runtime vtables. Imagine someone makes they own CustomModule : Module which is compiled against public api, the type won't load because it does not implement required GetImpl